src/README.md

```diff
@@ -413,6 +413,52 @@ void Initialize(Local<Object> target,
 NODE_MODULE_CONTEXT_AWARE_INTERNAL(cares_wrap, Initialize)
 ```
 
+If the C++ binding is loaded during bootstrap, it needs to be registered
+with the utilities in `node_external_reference.h`, like this:
+
+```cpp
+namespace node {
+namespace utils {
+void RegisterExternalReferences(ExternalReferenceRegistry* registry) {
+  registry->Register(GetHiddenValue);
+  registry->Register(SetHiddenValue);
+  // ... register all C++ functions used to create FunctionTemplates.
+}
+}  // namespace util
+}  // namespace node
+
+NODE_MODULE_EXTERNAL_REFERENCE(util, node::util::RegisterExternalReferences)
```

Comment:
Might want to also point out that `util` needs to be listed in the `node_external_reference.h` `EXTERNAL_REFERENCE_BINDING_LIST_BASE` macro